wolfcrypt/benchmark/benchmark.c

```diff
@@ -7163,6 +7174,13 @@ void bench_rsa(int useDeviceID)
 #else
         /* Note: To benchmark public only define WOLFSSL_PUBLIC_MP */
         rsaKeySz = 0;
+#endif
+#if defined(WOLFSSL_MICROCHIP_TA100)
+        /* Create new keys since you cannot import a private key to TA100 */
+        ret = wc_MakeRsaKey(&rsaKey[i], rsaKeySz, WC_RSA_EXPONENT, &gRng);
```

Comment:
This seems like an odd workaround. Does it even make sense to support this benchmark with a TA100?

Reply:
Still outstanding and not addressed. Please either defend the make rsa key workaround as a good addition or have an alternative.